corehq/apps/export/templates/export/commcare_analytics.html

```diff
@@ -0,0 +1,13 @@
+{%  extends 'hqwebapp/base_section.html' %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+{% load compress %}
+{% load hq_shared_tags %}
+
+{% block page_content %}
+  {% initial_page_data 'analytics_redirect_url' analytics_redirect_url %}
+  <p>Click the button below to navigate to the CommCare Analytics tool.</p>
+  <button class="btn btn-primary" type="button" onclick="window.open('{{ analytics_redirect_url }}', '_blank')">
```

Comment:
ux nitpick: links should generally just be links, not buttons. It's opening in a new window, but a lot of our links do.

Reply:
I used a button lest the page looks dull, but maybe that shouldn't be a UX guideline? The alternative is simply text that says something like "Navigate to Superset"...would that be OK?

Reply:
> maybe that shouldn't be a UX guideline

I see what you mean about the page being rather bare, but yeah, generally not 🙂  That text sounds good.

Reply:
Doesn't it feel like it's missing something? A title maybe (but most of the other views don't have titles)?

Maybe I'm completely overthinking this...
 
![image](https://user-images.githubusercontent.com/44245062/189909452-736c8f8b-af62-4471-9a84-bf7773320577.png)

Reply:
Hmm. It is aggressively minimal. I think it's okay, though.

If it's bugging you, some thoughts: Are there docs on superset you could link to from here? Or you could have a sentence saying what superset *is* (like the top of the mobile workers page, there's some lead text).